src/OrchardCore.Modules/OrchardCore.ContentFields/Settings/ContentPickerSettingType.cs

```diff
@@ -0,0 +1,7 @@
+namespace OrchardCore.ContentFields.Settings;
+public enum ContentPickerSettingType
+{
+    AllTypes,
+    ContentTypes,
+    Stereotypes
```

Comment:
I'm not sure if we can say `StereoTypes` in English

Reply:
`Stereotypes` is a single word